3rd_party/build-pytorch.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+# Build the pytorch libraries. The Git submodules must have been updated before
+# building. This Command need only be run once
+#  git submodule update --init --recursive
+
+mkdir pytorch/build_libtorch && cd pytorch/build_libtorch
+python ../tools/build_libtorch.py
```

Comment:
This should be incorporated into the build setup instructions I referred to in my other comment.  But on macOS, by default `python` will run Python 2.  If you intend Python 3 here then the earlier instructions in `macos.md` should say how to set up Python and its environment variables for this build.  On the reference build server we want the bare minimum steps to get to the build environment that will build our code.